### ISSUE TITLE
Start_frm is starting from 0. What is the criterion for ending?

### ISSUE THREAD

**Issue by 953585130** [facebookresearch/VMZ#68]
Hello there,
I've fine-tuned my own dataset, and then I want to use csv files to extract features for intensive prediction. The beginning of the csv file is org_video, label, start_frm, video_id; start_frm starts from 0, so how to determine the size of the end frame?
I hope to get your help, thank you very much!

**Comment by bjuncek** [comment]
You need to calculate that on your own I think. You can do that by using opencv capture FRAME_COUNT property. 
```
  int num_of_frames = cap.get(CV_CAP_PROP_FRAME_COUNT);
```

also, take a look here - might be useful
https://github.com/kenshohara/3D-ResNets-PyTorch/blob/master/utils/n_frames_kinetics.py

**Comment by dutran** [comment]
Thank Bruno!
